utils/mil_msgs/msg/LabeledImage.msg

```diff
@@ -0,0 +1,7 @@
+# Represents the complete data from a labeled image from LabelMe
+
+sensor_msgs/Image image            # Image where objects are labeled
```

Comment:
One problem with this is it is ambiguous what camera topic the labels are for. Is this for the rectified image or raw, etc.